util.js

```diff
@@ -0,0 +1,10 @@
+function checkOptions(actual, supported) {
+    const actualOptions = Object.keys(actual);
+    actualOptions.forEach(option => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9329266)
```suggestion
    actualOptions.forEach((option) => {
```